pom.xml

```diff
@@ -94,15 +106,51 @@
       <artifactId>hadoop-client-runtime</artifactId>
       <version>${hadoop.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>${log4j2.version}</version>
+    </dependency>
```

Comment:
Specifying the BOM in the dependencyManagement section eliminates the need to specify any versions for any of these log4j2 jars in this section.